lib/label_printer/label_printer/multiple_double_labels.rb

```diff
@@ -13,8 +13,9 @@ def double_label(asset)
         [label(asset), extra_label(asset)]
       end
 
+      # TODO: remove
       def extra_label(asset)
```

Comment:
Yep, similar to the label/create label stuff I mentioned earlier.

I was wondering if you could actually keep these around, and use them to DRY out the `label_name: 'main_label'` and `label_name: 'extra_label'` stuff. There's obviously a bit of repetition there, but I'm not sure if this would be an improvement or not.